dags/ft_donnees_brutes.py

```diff
@@ -0,0 +1,54 @@
+from airflow import DAG
+from airflow.decorators import task
+from airflow.operators import empty
+
+from dags.common import db, default_dag_args, slack
+
+
+with DAG(
+    "france_travail",
+    schedule_interval="@weekly",
+    **default_dag_args(),
+) as dag:
+    start = empty.EmptyOperator(task_id="start")
+
+    end = slack.success_notifying_task()
+
+    @task(task_id="store_ft")
+    def store_ft(**kwargs):
+        import ftputil
+        import pandas as pd
+
+        hostbucket, userbucket, passwordbucket = db.bucket_connection()
+
+        ft_raw_data = "raw_data_ft.csv"
+
+        db.drop_view("stg_france_travail")
+
+        with ftputil.FTPHost(hostbucket, userbucket, passwordbucket) as host:
+            # Check if the remote file exists
+            if host.path.isfile(ft_raw_data):
+                # Download the file to a local temporary file
+                with host.open(ft_raw_data, "rb") as ft_loaded_data:
+                    df = pd.read_csv(ft_loaded_data, sep="$")
+                    freins_cols = [
+                        "FREIN_ENTREE_FAMILLE",
+                        "FREIN_ENTREE_ADMIN",
+                        "FREIN_ENTREE_FINANCE",
+                        "FREIN_ENTREE_LOGEMENT",
+                        "FREIN_ENTREE_LANGUESAVOIR",
+                        "FREIN_ENTREE_MOBILITE",
+                        "FREIN_ENTREE_NUMERIQUE",
+                        "FREIN_ENTREE_SANTE",
+                        "FREIN_ENTREE_FORMATION",
+                    ]
+                    df[freins_cols] = df[freins_cols].replace("YYYY", None)
```

Comment:
Je suis nul en pandas mais c'est pas équivalent à ça ?
```suggestion
                    df[freins_cols].replace("YYYY", None, inplace=True)
```